### ISSUE TITLE
Extracting downloaded file goes forever

### ISSUE THREAD

**Issue by wanadam97** [stankovski/quran-phone#222]
Please help me how to fix this issue. Just download
<img width="601" alt="Screenshot 2021-03-11 124647" src="https://user-images.githubusercontent.com/78415529/110737184-db743300-8267-11eb-88d8-0dfb6e0f75cc.png">
ed from Microsoft store, downloaded required files and then nothing happens

**Comment by kbarnawi** [comment]
@stankovski yes, I have the same problem is there in fix for that I really like this app it just needs more contributor working on it same number of developers working on the android app. especially given the number off windows 10 machine growing very fast during this time of covid 19 and Ramadan is close
I am using the app on my PC

**Comment by DavDevPro** [comment]
> Please help me how to fix this issue. Just download <img alt="Screenshot 2021-03-11 124647" width="601" src="https://user-images.githubusercontent.com/78415529/110737184-db743300-8267-11eb-88d8-0dfb6e0f75cc.png"> ed from Microsoft store, downloaded required files and then nothing happens

**I have same error !**
